ChiModules/LBSCurvilinear/lbs_curvilinear_sweepchunk_pwl.cc

```diff
@@ -220,7 +224,7 @@ LBSCurvilinear::SweepChunkPWL::Sweep(chi_mesh::sweep_management::AngleSet* angle
               (face.normal.Dot(normal_vector_boundary) < -0.999999);
             if (!incident_on_symmetric_boundary)
             {
-              const int bndry_index = face.neighbor_id;
+              const uint64_t bndry_index = face.neighbor_id;
```

Comment:
good idea. for very large meshes, do you expect that we exceed the encoding afforded by simple `int` ?

Reply:
No. But face.neighbor_id has a dual purpose of holding either the neighbor cell index (when `has_neighbor=true`) or the boundary index.